skins/laika/src/scss/variables.scss

```diff
@@ -39,6 +39,10 @@ $custom-colors: (
 		"gray-very-light": ($fun-color-gray-very-light),
 );
 
+// 320px is the smallest assumed mobile size, minus 2 x column gap (18px).
+// We don't use calc(100vw-36px) for compat reasons
+$fun-smallest-width: 284px;
```

Comment:
I think according to the mobile design, there should be a minimum 3 pixels gap on the left and right of the screen, so the white main form does not extend all the way to the sides of the screen on mobile. This means the form is 314px wide in its smallest state rather than 320px, so this value should be 278px.

Reply:
Good catch! I'm thanking my past self for making this a variable.